webauthn-rp-proxy/src/main.rs

```diff
@@ -0,0 +1,202 @@
+use clap::{App, Arg};
+use serde::{Deserialize, Serialize};
+use serde_json::Result;
+use std::io::{self, Read};
+use uuid::Uuid;
+use webauthn_rs::prelude::*;
+
+// A simple command-line wrapper around webauthn-rs
```

Comment:
If you make these rustdoc module comments (lines starting with `//!`), then this information will show up in the built documentation (`cargo doc`) and rendered in markdown.

Reply:
Done.

Reply:
Yep, I've written https://github.com/kanidm/webauthn-rs/pull/426 to fix that up.

You probably want to rebase your PR on that and have a look at the output, because don't think the formatting has come out the way you intended.

Reply:
Fixed.  Thanks.

Sorry for all the extra Git traffic.  I believe I've rebased on top of origin/master now, so the diffs are correct.

Reply:
No worries :)